modules/ROOT/pages/bat-command-reference.adoc

```diff
@@ -4,12 +4,35 @@ The BAT command line interface lets you run tests locally and schedule monitors
 
 This reference covers the syntax and key options you can use with the *bat* command. Using the *bat* command, you can debug and run tests, get credentials for scheduling tests, schedule tests, list test schedules, and perform additional operations.
 
+For commands to execute successfully, the account under which the BAT CLI logs in must have the required permissions for the action the command is executing. 
+
+== Permissions Required
+
+To use all of the BAT commands to execute the corresponding actions in Anypoint Platform, you must have the following permissions. 
+
+[options="header,footer"]
+|=======================
+|Permission |Required For
+|Create Applications (Runtime Manager) |Create private locations
+|Delete Applications (Runtime Manager) |Delete private locations
+|Manage Settings (Runtime Manager) |Manage private locations
+|View Environment |Log in using BAT CLI
+|View Organization |Log in through a connected app using BAT CLI
```

Comment:
This one is used for internal use from secrets manager team to check the org entitlements. So it's only needed for Use secrets manager through BAT CLI